data/covering_grammar/lib/make_test_file.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Makes test file.
+
+Takes the gold data and the model output, and creates a three-column TSV where
+each line has a word, its gold pronunciation, and the predicted pronunciation.
+Assumes that the input files have the same words in the same order.
+"""
+
+import argparse
+import logging
+
+
+def main(args: argparse.Namespace) -> None:
+    with open(args.gold, "r") as gf, open(args.pred, "r") as pf:
+        with open(args.out, "w") as wf:
+            for i, (g_line, p_line) in enumerate(zip(gf, pf)):
+                try:
+                    g_data = g_line.split("\t")
```

Comment:
Why don't you unpack these here? E.g., `(g_pron, g_word)`?

Indexing into a small tuple like you do later is code smell, to me.